src/main/java/beam/analysis/plots/RealizedModeAnalysis.java

```diff
@@ -625,6 +630,56 @@ public void writeToReferenceCSV(){
         }
     }
 
+    private void writeToReplaningChainCSV(IterationEndsEvent event) {
+        String fileName = GraphsStatsAgentSimEventsListener.CONTROLLER_IO.getIterationFilename(event.getIteration(), "replanningEventChain.csv");
+        String REPLANNING_SEPARATOR = "-"+ReplanningEvent.EVENT_TYPE+"-";
+        try (BufferedWriter out = new BufferedWriter(new FileWriter(new File(fileName)))) {
```

Comment:
This will not close the `FileWriter` resource afaik. You'd have to double try or use an explicit finally I believe.

Reply:
used double try to close both streams individually